packages/netlify-cms-widget-media/src/index.js

```diff
@@ -0,0 +1,15 @@
+import withFileControl from './withFileControl';
```

Comment:
How is this different from the file widget?

Reply:
Could you please rephrase this? I am not sure I understand.

Reply:
I think it might be better to use the term `media` in the code instead of file/image.
That would require modifying the widget code and doing some cleanup.